src/web/components/Core/TermsAndConditions.tsx

```diff
@@ -11,233 +237,16 @@ export type AcceptTermForm = {
   acceptConditions: boolean;
 };
 
-export function TermsAndConditions({ onAccept, onCancel }: TermsAndConditionsProps) {
+export function TermsAndConditionsForm({ onAccept, onCancel }: TermsAndConditionsFormProps) {
```

Comment:
why do we call this a Form? I didn't see any submit action?

Reply:
Because there's a button at the bottom - but it's called Accept, not Submit.